docs/source/posts/2024/2024-08-21-wachiou-final-report.rst

```diff
@@ -0,0 +1,282 @@
+         ]
+
+
+   2. Investigating and Fixing the Footer Deformation Issue (https://github.com/fury-gl/fury/issues/874 (closed)):
+
+      - Issue Identification: The footer on the FURY website deformed when hovering over elements due to size increases, which affected the padding of its container and the layout of subsequent elements.
+
+      - Initial Approach: Bold Styling: Attempted to resolve the issue by making the elements bold on hover instead of changing font size. While this approach fixed the deformation, it did not meet the design requirements for the homepage footer.
+
+      - Final Fix: CSS Adjustments: Added properties to the ``.class-columns`` in ``styles.css`` to better manage the footer style and prevent layout issues, ensuring that the design integrity was maintained.
+
+      Video Demonstrations:
+      Before Fixing the Footer Issue:
+
+      Video demonstrating the footer deformation before the fix.
+
+      .. raw:: html
+
+         <iframe src="https://github.com/user-attachments/assets/2f5d4021-b661-4be9-944f-7a2638376f2c" width="640" height="390" frameborder="0" allowfullscreen></iframe>
+
+
+
+      After Fixing the Footer Issue:
+
+      Video showing the footer after applying the fix.
+
+      .. raw:: html
+
+         <iframe src="https://github.com/user-attachments/assets/b0ec74df-827b-4280-b3e4-bf968b97a654" width="640" height="390" frameborder="0" allowfullscreen></iframe>
+
+
+
+*Pull Requests*:
+
+- Addressed Sphinx warnings and updated configuration: https://github.com/fury-gl/fury/pull/922 (merged)
+- https://github.com/fury-gl/fury/pull/915 (merged)
+- fix typos in blog posts: https://github.com/fury-gl/fury/pull/911 (merged)
+- Fix footer deformation issue: https://github.com/fury-gl/fury/pull/925 (merged)
+
+Other Objectives
+----------------
+
+1. Peer Review Contributions:
+   - Reviewed and provided feedback on PRs from other contributors to the FURY and DIPY projects.
+   - Assisted in resolving issues and improving code quality in the PRs.
+
+*Pull Requests*:
+
+- https://github.com/fury-gl/fury/pull/913 (merged)
+
+
+Objectives in Progress
+----------------------
+
+1. Enhancing Documentation:
+   - Ongoing work to improve the documentation structure and content.
+   - Addressing additional Sphinx warnings.
+
+2. Modernizing Codebase:
+   - Refactoring existing code to align with the latest Python standards.
+   - Implementing best practices for code maintainability and readability.
+   - Separating FURY's codebase and website.
+
+3. Community Engagement:
+   - Engaging with the FURY community through discussions, feedback, and contributions.
+   - Participating in community events and meetings to share progress and gather insights.
+
+
+GSoC Weekly Blogs
+-----------------
+
+- My blog posts can be found at `FURY website <https://fury.gl/latest/blog/author/wachiou-bouraima.html>`_ and `Python GSoC Mastodon server <https://social.python-gsoc.org/@wasscodeur>`_.
+
+Timeline
+--------
+
```

Comment:
Use `list-table` directive if possible, it's simpler. Have a look at Robin's report.

Reply:
Thank you